docs/available-filters/envelope-encryption/hashicorp-vault/setup.adoc

```diff
@@ -0,0 +1,115 @@
+:hashicorp-vault: https://developer.hashicorp.com/vault
+
+= HashiCorp Vault
+
+== Vault Service URL
+
+The Vault Service URL is required so the filter knows how to connect to Vault. This address is reported by Vault as the
+`Api Address` as it {hashicorp-vault}/tutorials/getting-started/getting-started-dev-server#starting-the-dev-server[starts up].
+
+== Enable the Transit Secrets engine
+
+The filter integrates with the {hashicorp-vault}/docs/secrets/transit[HashiCorp Vault *transit
+secrets engine*].   Vault does not enable the transit secrets engine by default.  It must be
+{hashicorp-vault}/docs/secrets/transit#setup[enabled] before it can be used with the filter.
+
+The transit engine's path must be `/transit` (the default).
```

Comment:
This is changing with https://github.com/kroxylicious/kroxylicious/pull/952 correct?

Reply:
Done